src/logistration/_style.scss

```diff
@@ -21,8 +21,6 @@ $apple-focus-black: $apple-black;
 
 .register-container {
   @extend .login-container;
-
-  width: 30rem;
```

Comment:
if login-container is not used anywhere else can we remove it and just add the values directly to register-container?

Reply:
@zainab-amir Actually both are not being used, I'm gonna remove both.